tests/unittest/_torch/modules/mamba/test_causal_conv1d_triton.py

```
@@ -0,0 +1,200 @@
+    total_tokens = sum(seq_lens)
+    # channel-last: x is (dim, cu_seqlen) with stride(0) == 1
+    x = torch.randn(total_tokens, dim, dtype=dtype, device=device, generator=generator).transpose(
+        0, 1
+    )
+    conv_state = torch.randn(
+        num_cache_lines, dim, state_len, dtype=dtype, device=device, generator=generator
+    )
+    initial_conv_state = conv_state.clone()
+
+    query_start_loc = torch.tensor(
+        [0] + torch.tensor(seq_lens).cumsum(0).tolist(), dtype=torch.int32, device=device
```

Ruff (0.16.2)

[warning] Consider `[0, *torch.tensor(seq_lens).cumsum(0).tolist()]` instead of concatenation

Replace with `[0, *torch.tensor(seq_lens).cumsum(0).tolist()]`

(RUF005)